### DISCUSSION TITLE
How to add extra scopes to generated provider files

### DISCUSSION THREAD

**Discussion by kthhrv** [GoogleCloudPlatform/cloud-foundation-fabric#1989] [Q&A]
I'm creating BQ tables with google sheets as their source and this requires the provider file have added scope "https://www.googleapis.com/auth/drive" where in the CFF would I make this change?

Thanks

**Comment by juliocc** [comment]
I understand that you're referring to the provider files generated by FAST.

If that's the case, you can add the [`scopes`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference#scopes) option to the generated files by updating [this template](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/fast/assets/templates/providers.tf.tpl)

**Reply by kthhrv** [reply]
perfect, thanks!
